src/Microsoft.ML.AutoML/Experiment/Runners/RunnerUtil.cs

```diff
@@ -48,6 +49,12 @@ public static (ModelContainer model, TMetrics metrics, Exception exception, doub
                 logger.Error($"Pipeline crashed: {pipeline.ToString()} . Exception: {ex}");
                 return (null, null, ex, double.NaN);
             }
+            finally
+            {
+                // Free Tensor objects in model. Tensor objects made in TensorFlow's C
+                // libraries are not automatically cleaned up by C#'s Garbage Collector.
+                (estimatorModel as IDisposable)?.Dispose();
```

Comment:
What occurs when the calling function tries to use the wrapped-and-returned-model? Unless the TF components are recreated behind the scenes, I would expect a null reference or use-after-free.